apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/RestResourceBase.java

```diff
@@ -44,6 +43,13 @@ public abstract class RestResourceBase {
   private static final Function<String, Uni<? extends Schema.CqlKeyspaceDescribe>>
       MISSING_KEYSPACE = ks -> Uni.createFrom().nullItem();
 
+  private static final Function<String, Uni<? extends QueryOuterClass.Response>>
+      MISSING_KEYSPACE_RESPONSE =
+          ks -> {
```

Comment:
Instead of throwing you should do 

`return Mono.createFrom().failure(exception)`